include/afl-fuzz.h

```diff
@@ -410,6 +410,19 @@ struct foreign_sync {
 
 };
 
+struct fuzz_event {
+  int64_t dur;
+  u64     original_cksum;
+  u64     fuzzed_cksum;
+};
+
+struct fuzz_event_buffer {
+  FILE *            log_file;
+  struct fuzz_event events[16384];
```

Comment:
Why not dynamically malloc and realloc? Pretty sure, you run out of 0x4000 entries in a matter of seconds.

Reply:
@mboehme Yeah, the buffer isn't very big. It was just a quick way to avoid too frequent I/O, but I could probably allocate a larger buffer on the heap instead.